src/runtime_src/doc/toc/cloud_vendor_support.rst

```diff
@@ -191,3 +191,14 @@ The following picture illustrates how XRT is being deployed in different scenari
 
 .. image:: xrt-deployment-cloud.svg
    :align: center
+
+Special Case
+============
+
+There is special case where download xclbin from within user VM is not required.
+
+In this special case, neither MSD/MPD nor plugins are required since there is no real xclbin download happening. xclbins can be preloaded
```

Comment:
there is no real xclbin download happening --> no xclbin download is allowed from guest.

Reply:
changed